en/programs/oie/Content/Topics/Settings/end-user-okta-account-chooser.htm

```diff
@@ -1,123 +1,10 @@
-        <meta name="robots" content="noindex, nofollow" />
-        <meta name="typeofcontent" content="documentation" />
-        <meta name="audience" content="admin" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/okta_help.htm?type=eu&amp;id=csh-user-plugin-switch-accounts" /><title>end-user-okta-account-chooser</title>
-        <link href="../../../Skins/Default/Stylesheets/Slideshow.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/TextEffects.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Topic.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Default/Stylesheets/Components/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/foundation.6.2.3.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Styles.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Tablet.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../../Skins/Fluid/Stylesheets/Mobile.css" rel="stylesheet" type="text/css" data-mc-generated="True" />
-        <link href="../../Resources/Stylesheets/okta-main-new-nav.css" rel="stylesheet" type="text/css" />
-        <script src="../../../Resources/Scripts/jquery.min.js" type="text/javascript">
-        </script>
-        <script src="../../../Resources/Scripts/purify.min.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/require.min.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/require.config.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/foundation.6.2.3_custom.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/plugins.min.js" type="text/javascript" defer="defer">
-        </script>
-        <script src="../../../Resources/Scripts/MadCapAll.js" type="text/javascript" defer="defer">
-        </script>
+        <meta name="robots" content="noindex, nofollow" />
+        <meta http-equiv="REFRESH" content="0.0;url=https://help.okta.com/okta_help.htm?type=eu&amp;id=csh-user-plugin-switch-accounts" />
```

Comment:
How did we find the alias for each page? Is there a map somewhere?
For example, if I navigate directly to https://help.okta.com/en/programs/oie/Content/Topics/Settings/end-user-okta-account-chooser.htm I see the "Switch between accounts" page,
but if I navigate to the new redirect, https://help.okta.com/okta_help.htm?type=eu&amp;id=csh-user-plugin-switch-accounts, then I see the end user overview page, so it is not the same

Reply:
That indeed looks like a busted alias. The aliases are maintained here: [https://github.com/okta/info-dev/blob/infodev/Project/Advanced/Aliases.flali](https://github.com/okta/info-dev/blob/infodev/Project/Advanced/Aliases.flali). 
We need a writer to fix the alias in `info-dev` source. I'll add a comment to our ticket about this.
Once the alias is updated in source and we refresh the End User publication, then this redirect link should work as expected.